Formula/flake8.rb

```diff
@@ -37,26 +38,10 @@ class Flake8 < Formula
   end
 
   def install
-    venv = virtualenv_create(libexec, "python3")
-    resource("entrypoints").stage do
-      # Without removing this file, `pip` will ignore the `setup.py` file and
-      # attempt to download the [`flit`](https://github.com/takluyver/flit)
-      # build system.
-      rm_f "pyproject.toml"
-      venv.pip_install Pathname.pwd
-    end
-    (resources.map(&:name).to_set - ["entrypoints"]).each do |r|
-      venv.pip_install resource(r)
-    end
-    venv.pip_install_and_link buildpath
+    virtualenv_install_with_resources
```

Comment:
@iMichka is this still correct for 3.8?

Reply:
Yes this will work, it should auto-detect python 3.8. I had no issues with this.